connector.go

```diff
@@ -89,6 +91,12 @@ func NewConnector(options ...Option) (*Connector, error) {
 	go http.ListenAndServe("0.0.0.0:8080", c.Health)
 	log.Info().Str("addr", "0.0.0.0:8080").Msg("healthcheck listening on /live and /ready")
 
+	//	Initialize trace provider
+	_, err = monitor.InitTracerProvider(context.TODO(), conf.GetString("trace.host.grpc"), c.id(), c.manifest.Version.String(), string(c.env), conf.GetFloat64("trace.sample.ratio"))
```

Comment:
do we want monitoring for all connectors in all environments? Do we want to allow devs to disable monitoring if they're just running it locally in dev environment?

Reply:
It won't crash when running locally without `opentelemetry`.